docs/topics/readme/configmap-README.adoc

```diff
@@ -1,13 +1,13 @@
-= ${mission} - ${runtime} Booster
+= ${mission} - ${runtime} Example Application
 
-IMPORTANT: While you can run and interact with this booster on localhost, you cannot take advantage of using externalized configuration with this booster without using OpenShift. For more details on using this booster with a single-node OpenShift cluster, CI/CD deployments, as well as the rest of the runtime, see the link:${guideURL}[${runtime} Runtime Guide].
+IMPORTANT: While you can run and interact with this example application on localhost, you cannot take advantage of using externalized configuration with this example application without using OpenShift. For more details on using this example application with a Single-node OpenShift cluster, CI/CD deployments, as well as the rest of the runtime, see the link:${guideURL}[${runtime} Runtime Guide].
```

Comment:
Single-node OpenShift cluster --> Single-node OpenShift Cluster
Not sure why the attributes are not used for this in the README content. Writing the name manually across 100-odd files might result in such discrepancies. Consider using the attributes if time permits.

Reply:
@snelluli1 thank you for pointing this out. The reason why don't substitute some terms using attributes in the README files is because these files were originally designed to be partially procedurally generated by the Launcher for the dual purpose of being displayed on-screen when creating an example application, as well as to be included in the ZIP filed with packaged example application projects. Because processing the README files with AsciiDoctor was not feasible in this case, we opted to use a custom BASH script that substituted entire source lines using shell variables.

Reply:
@snelluli1 @huntley The above being said, I'd like to point out that we should no longer use the term _Single -node OpenShift Cluster_. We will be replacing it with _OpenShift cluster_ for [NON-PUBLIC PRODUCT RELATED REASONS]